### DISCUSSION TITLE
Kong ingress issue: error="posting new config to /config: HTTP status 400 (message: \"declarative config is invalid: {plugins={{config={header_type=\\\"expected one of: preserve, b3, b3-single, w3c, jaeger, ot\\\"}}}}\")" subsystem=proxy-cache-resolver

### DISCUSSION THREAD

**Discussion by guptadeepali39** [Kong/kong#8217] [Help] [answered]
Hi,

I am installing my kong using latest hem chart, but  my application's ingress is not taking up the address of kong ingress controller.
When I checked the logs of kong ingress controller, I found a continuous error poping up:

time="2021-12-19T22:22:31Z" level=error msg="could not update kong admin" error="posting new config to /config: HTTP status 400 (message: \"declarative config is invalid: {plugins={{config={header_type=\\\"expected one of: preserve, b3, b3-single, w3c, jaeger, ot\\\"}}}}\")" subsystem=proxy-cache-resolver

I am unable to find what is the issue.

Can someone please guide.

**Comment by kikito** [comment, marked as answer]
There seems to be a problem on the field `config.header_type` of your Zipkin plugin. What's your current configuration for that plugin?

**Reply by guptadeepali39** [reply]
Hi @kikito thanks for pointing out. There was issue with zipkin plugin , I redeployed it. It worked, thanks alot.
